public/js/directives/course/course-edit-modal.html

```diff
@@ -1,5 +1,6 @@
 <div class="course-edit-modal">
     <div class="modal-header">
+        <button type="button" class="close" ng-click="cancel()" aria-label="Close"><span aria-hidden="true">&times;</span></button>
```

Comment:
You could use the fa icon fa-times instead of this, if you prefer. Or even just the letter X